Gems/Multiplayer/Code/Source/Components/NetworkTransformComponent.cpp

```diff
@@ -44,7 +44,7 @@ namespace Multiplayer
         GetNetBindComponent()->AddEntityCorrectionEventHandler(m_entityCorrectionEventHandler);
         ParentEntityIdAddEvent(m_parentChangedEventHandler);
 
-        if (!HasController())
+        if (!GetNetBindComponent()->IsNetEntityRoleAuthority())
```

Comment:
Is this not on Authority because Authority already has this information?

Reply:
that's correct. The hierarchies are formed in the Authority, so we don't need to run that code.